mappings/net/minecraft/world/gen/feature/IcebergFeature.mapping

```diff
@@ -2,11 +2,21 @@ CLASS net/minecraft/class_3074 net/minecraft/world/gen/feature/IcebergFeature
 	METHOD method_13414 isAirBelow (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
 		ARG 1 world
 		ARG 2 pos
-	METHOD method_13418 (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;IIZI)V
+	METHOD method_13415 carve (IILnet/minecraft/class_2338;Lnet/minecraft/class_1936;ZDLnet/minecraft/class_2338;II)V
+	METHOD method_13416 getEllipseCurve (III)I
+	METHOD method_13417 heightDependentEllipseRadius (III)I
+	METHOD method_13418 smooth (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;IIZI)V
 		ARG 1 world
 		ARG 2 pos
+	METHOD method_13419 heightDependentRoundRadius (Ljava/util/Random;III)I
 	METHOD method_13420 isSnowyOrIcy (Lnet/minecraft/class_2248;)Z
 		ARG 1 block
+	METHOD method_13421 signedCircleDistance (IILnet/minecraft/class_2338;ILjava/util/Random;)D
 	METHOD method_13422 clearSnowAbove (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)V
 		ARG 1 world
 		ARG 2 pos
+	METHOD method_13424 signedEllipseDistance (IILnet/minecraft/class_2338;IID)D
+	METHOD method_13425 setIcebergBlock (Lnet/minecraft/class_2338;Lnet/minecraft/class_1936;Ljava/util/Random;IIZZLnet/minecraft/class_2680;)V
+	METHOD method_13426 generateIcebergBlock (Lnet/minecraft/class_1936;Ljava/util/Random;Lnet/minecraft/class_2338;IIIIIIZIDZLnet/minecraft/class_2680;)V
+	METHOD method_13427 heightDependentSteepRadius (Ljava/util/Random;III)I
+	METHOD method_13428 generateCutOut (Ljava/util/Random;Lnet/minecraft/class_1936;IILnet/minecraft/class_2338;ZIDI)V
```

Comment:
please see my other comments about verb phrases and verbosity, and lack of info.